src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -126,7 +126,9 @@ export const TimelineNav = ({
     return allByType(children, TimelineNavPanel);
   }, [children]);
 
-  const timelineNavItemRefs = timelineNavItems().map(() => React.createRef());
+  const timelineNavItemRefs = timelineNavItems().map(() =>
+    React.createRef<HTMLAnchorElement>(),
```

Comment:
I'm not convinced this is good use of `React.createRef`, since it's creating a new ref on every render (which sort of defeats the purpose of a ref). Will investigate.